src/test/java/org/junit/tests/running/classes/ThreadsTest.java

```diff
@@ -0,0 +1,86 @@
+package org.junit.tests.running.classes;
+
+import org.junit.AfterClass;
+import org.junit.Test;
+import org.junit.internal.runners.ErrorReportingRunner;
+import org.junit.runner.*;
```

Comment:
For newer code we avoid wildcard imports.

Reply:
Done. IDE behavior :disappointed: